CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 **Features**:
 
+- Compilation options can now be specified from Python. (@eitsupi, #1807)
```

Comment:
Ideally, it would be necessary to make it easier to understand which subproject the change relates to, but so far nothing has been done.

Reply:
I was thinking we'd put these in the "Integrations" (which maybe should be "Integrations / Bindings" section. I don't think we'll have enough that we'll need multiple ones... And then the "Features" section would be for language features.

What do you think?

Reply:
Sorry, I missed the integration section...